pyproject.toml

```diff
@@ -202,6 +202,7 @@ git-only = [
   ".coveragerc",
   ".dockerignore",
   "shell.nix",
+  "mypy.ini",
```

Comment:
fyi mypy configuration is already in the pyproject.toml file (see line 258), the syntax is just a little bit different https://mypy.readthedocs.io/en/stable/config_file.html